packages/reports/addon/components/filter-values/lookback-input.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   {{filter-values/lookback-input
+ *       filter=filter
+ *       request=request
+ *       onUpdateFilter=(action 'update')
+ *   }}
+ */
+import Component from '@ember/component';
+import { layout as templateLayout, classNames } from '@ember-decorators/component';
+import layout from '../../templates/components/filter-values/lookback-input';
+import { computed, get, action } from '@ember/object';
+import Interval from 'navi-core/utils/classes/interval';
+import { isEmpty } from '@ember/utils';
+import { formatDateRange } from 'navi-reports/helpers/format-interval-inclusive-inclusive';
+import { getIsoDateTimePeriod } from 'navi-core/utils/date';
+
+@templateLayout(layout)
+@classNames('filter-values--lookback-input')
+class LookbackInput extends Component {
+  /**
+   * @property {String} timeGrainName - The dateTimePeriod
+   */
+  @computed('request.logicalTable.timeGrain.name')
+  get timeGrainName() {
+    return get(this, 'request.logicalTable.timeGrain.name');
+  }
+
+  /**
+   * @property {Interval} interval - The current interval
+   */
+  @computed('filter.values.firstObject')
+  get interval() {
+    return get(this, 'filter.values.firstObject');
```

Comment:
This seems to be a pattern with these filter value components. Maybe a base class would be good?